packages/manager/modules/netapp/src/components/select-checkboxes/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,5 @@
+{
+  "select_checkboxes_placeholder_default": "Choisissez une option",
+  "select_checkboxes_one_item": "1 item",
```

Comment:
**questions:** Content has been validated as is for french?

Reply:
No content for the component itself in case of different kind of items. The page itself will override these defaults with the content to be used.